connexion/api.py

```diff
@@ -40,12 +41,16 @@ def compatibility_layer(spec):
     # Make all response codes be string
     for path_name, methods_available in spec.get('paths', {}).items():
         for method_name, method_def in methods_available.items():
-            if method_name == 'parameters':
+            if method_name == 'parameters' or \
+               not isinstance(method_def, dict):
```

Comment:
If you really want to know what happened for that line of code to be there you can look into the git history (git blame/etc) instead of polluting the code with many comments pointing to issue numbers.